chart/templates/_helpers.tpl

```diff
@@ -139,13 +224,38 @@ The URL to access the admin service from another container
 See https://kubernetes.io/docs/concepts/services-networking/dns-pod-service/#a-aaaa-records
 */}}
 {{- define "admin.url" -}}
-{{- printf "http://%s-admin.%s.svc.cluster.local:80" ( include "release" . ) ( .Release.Namespace ) }}
+{{- printf "http://%s-admin.%s.svc.cluster.local:80" ( include "name" . ) ( .Release.Namespace ) }}
 {{- end }}
 
 {{/*
 The URL to access the API service from another container
 See https://kubernetes.io/docs/concepts/services-networking/dns-pod-service/#a-aaaa-records
 */}}
 {{- define "api.url" -}}
-{{- printf "http://%s-api.%s.svc.cluster.local:80" ( include "release" . ) ( .Release.Namespace ) }}
+{{- printf "http://%s-api.%s.svc.cluster.local:80" ( include "name" . ) ( .Release.Namespace ) }}
```

Comment:
You changed the port to 8080 in values.yaml, shouldn't you change here too?

Reply:
Fixed